quarkus/tests/integration/src/main/java/org/keycloak/Keycloak.java

```diff
@@ -162,20 +171,32 @@ private ApplicationModel createApplicationModel(String keycloakVersion)
 
     private WorkspaceModule createWorkspaceModule(String keycloakVersion) {
         Path moduleDir = createModuleDir();
+        DependencyBuilder serverDependency = DependencyBuilder.newInstance()
+                .setGroupId("org.keycloak")
+                .setArtifactId("keycloak-quarkus-server-app")
+                .setVersion(keycloakVersion)
+                .addExclusion("org.jboss.logmanager", "log4j-jboss-logmanager");
+
+        if (fipsEnabled) {
+            serverDependency.addExclusion("org.bouncycastle", "bcprov-jdk15on");
+            serverDependency.addExclusion("org.bouncycastle", "bcpkix-jdk15on");
```

Comment:
Should `keycloak-crypto-default` be excluded too?

Reply:
Fixed.